### ISSUE TITLE
[master branch] db scripts are broken - "c.skin" sql error on a fresh install

### ISSUE THREAD

**Issue by shdr212** [TrinityCore/TrinityCore#25740]
**Description:**

Seems like currently the db scripts are broken.
I created a new database, created the basic database and user ( https://github.com/TrinityCore/TrinityCore/blob/master/sql/create/create_mysql.sql ), and then ran the worldserver with auto updater.

It took a bit but then it showed: 
`ERROR 1054 (42S22) at line 8542: Unknown column 'c.skin' in 'on clause'`

**Expected behaviour:**

Database set up correctly on a fresh install

**Steps to reproduce the problem:**

1. Setup a new database
2. Apply the initial creation script (create_mysql.sql)
3. Run worldserver with auto updater enabled

**Branch(es):**

master

**TC rev. hash/commit:** 

Commit 8a21f607a8c661abaae8dcbf813080ab9d7c5ae8

**Operating system:**

debian 10

**Entire log from worldserver:**
```
ERROR 1054 (42S22) at line 8542: Unknown column 'c.skin' in 'on clause'

Applying of file '/srv/sql/updates/characters/master/2020_11_16_00_characters.sql' to database 'characters' failed! If you are a user, please pull the latest revision from the repository. Also make sure you have not applied any of the databases with your sql client. You cannot use auto-update system and import sql files from TrinityCore repository with your sql client. If you are a developer, please fix your sql query.
Could not update the Character database, see log for details.
TrinityCore rev. 8a21f607a8c6 2020-12-12 22:04:47 +0100 (master branch) (Unix, RelWithDebInfo, Static) (worldserver-daemon)
<Ctrl-C> to stop.

 ______                       __
/\__  _\       __          __/\ \__
\/_/\ \/ _ __ /\_\    ___ /\_\ \, _\  __  __
   \ \ \/\`'__\/\ \ /' _ `\/\ \ \ \/ /\ \/\ \
    \ \ \ \ \/ \ \ \/\ \/\ \ \ \ \ \_\ \ \_\ \
     \ \_\ \_\  \ \_\ \_\ \_\ \_\ \__\\/`____ \
      \/_/\/_/   \/_/\/_/\/_/\/_/\/__/ `/___/> \
                                 C O R E  /\___/
http://TrinityCore.org                    \/__/

Using configuration file /srv/worldserver.conf.
Using SSL version: OpenSSL 1.1.1d  10 Sep 2019 (library: OpenSSL 1.1.1d  10 Sep 2019)
Using Boost version: 1.67.0
Updating Auth database...
>> Auth database is up-to-date! Containing 3 new and 177 archived updates.
Updating Character database...
>> Reapplying update "2020_11_16_00_characters.sql" '33D5C75' -> 'DE33D26' (it changed)...
```

**Comment by illfated** [comment]
https://github.com/TrinityCore/TrinityCore/blob/8a21f607a8c661abaae8dcbf813080ab9d7c5ae8/sql/updates/characters/master/2020_11_16_00_characters.sql#L8541-L8560

**Comment by Shauren** [comment]
For some reason your core thinks that 2020_11_16_00_characters.sql has a different hash than `33D5C75` and tries to reapply it - but on a clean clone in a linux virtual machine it shows `33D5C75` to me, did you modify that file in some way?

**Comment by Aokromes** [comment]
33d5c7539e239132923d01f4b6ead5f3ef3eeb69  sql/updates/characters/master/2020_11_16_00_characters.sql
for me.

**Comment by shdr212** [comment]
@Shauren Nope I didn't modify it.
How can I calculate the file hash my self, like @Aokromes is showing?

**Comment by Shauren** [comment]
`sha1sum sql/updates/characters/master/2020_11_16_00_characters.sql`

**Comment by Aokromes** [comment]
> @Shauren Nope I didn't modify it.
> How can I calculate the file hash my self, like @Aokromes is showing?
> 
> p.s.
> I also looked at my source directory again and ran `git status .`
> If the file was modified, it should of shown it, which it doesn't.

i suggest you to fresh clone repo again.

**Comment by shdr212** [comment]
Ha I think I found the issue; it has something to do with newlines settings and git.
It's also the reason why git doesn't show any diff content.

I'll verify it and close this repo if that is truly the case.

**Comment by shdr212** [comment]
Ok so the line ending were the problem.

However I believe it is a problem. In my current setup I checked out the files on windows, then copied them to a linux machine.

When checked out in windows, cr were replaced by crlf.
I tried to set `core.autocrlf` to false on the repository after clone, but no matter what I did git still checked them out with crlf (can be seen with `git ls-files --eol`).

To bypass it I ran dos2unix on all the .sql files, but a better solution should be made.
